index.js

```diff
@@ -175,12 +176,18 @@ Funnel.prototype.build = function() {
   var linkedRoots = false;
   if (this.shouldLinkRoots()) {
     linkedRoots = true;
-    if (fs.existsSync(inputPath)) {
+
+    var inputPathExist = fs.existsSync(inputPath);
+
+    if (this._lastInputPath ? !inputPathExist : inputPathExist) {
       rimraf.sync(this.outputPath);
-      this._copy(inputPath, this.destPath);
-    } else if (this.allowEmpty) {
-      mkdirp.sync(this.destPath);
```

Comment:
I believe `if (!inputPathExists && this.allowEmpty) { mkdirp.sync(this.destPath); }` should be restored, this addresses one of the failures. "does not error with input node at a missing nested source"